aspnet/security/app-secrets.rst

```diff
@@ -1,9 +1,11 @@
 Safe Storage of Application Secrets
 ===================================
 
-By `Rick Anderson`_
+By `Rick Anderson`_, `Daniel Roth`_
 
-This tutorial shows how your application can securely store and access secrets in the local development environment. The most important point is you should never store passwords or other sensitive data in source code, and you shouldn't use production secrets in development and test mode. The secret manager tool was written to help prevent sensitive data from being checked into source control. The :doc:`/fundamentals/configuration` system that is used by default in DNX based apps can read secrets stored with the secret manager tool described in this article.
+Apps frequently require secrets, such as a client ID for OAuth or database connection strings with credentials. These passwords and other sensitive data should never be kept in source code or added to configuration files inside your project's source tree, as they can easily be accessed by untrusted parties.
```

Comment:
I like the original better. The new version is missing the "you shouldn't use production secrets in development and test mode." - this isn't directly related to the secrets manager, but we should help our customers fall into the pit of security success. @blowdart actually came up with the original. "The most important point" is attention grabbing.